tests/scripts/tools.lua

```diff
@@ -412,8 +412,8 @@ function drawing_with_symmetry(imageColorMode, colorInImage,
   expect_eq(cel.bounds, Rectangle(0, 0, 8, 3))
   expect_img(cel.image,
              { 0, 0, 0, 0, 0, 0, 0, 0,
-	       0, 0, 0, 0, 0, 0, 0, 0,
-	       0, 0, 0, 0, 0, 0, 0, 0 })
+	             0, 0, 0, 0, 0, 0, 0, 0,
+	             0, 0, 0, 0, 0, 0, 0, 0 })
```

Comment:
Remove this change in indentation in the whole file, try to use whitespaces instead of tabs.